packages/edit-site/src/components/dataviews/dataviews.js

```diff
@@ -158,6 +159,25 @@ export default function DataViews( {
 			<VStack spacing={ 4 }>
 				<HStack justify="space-between">
 					<TextFilter onChange={ dataView.setGlobalFilter } />
+					{ Object.keys( view.filters ).map( ( key ) => {
+						const field = dataView
```

Comment:
We should avoid using `dataView` (the tan stack APIs) here, instead we should use the `fields` prop. The reason is simple, filters are cross view type (layout), and tan stack is specific to the table.